mailpoet/lib/Services/Bridge/API.php

```diff
@@ -7,6 +7,10 @@
 use WP_Error;
 
 class API {
+  const AUTHORIZED_EMAIL_STATUS_OK = 'ok';
+  const AUTHORIZED_EMAIL_STATUS_ERROR = 'authorized_email_error';
+  const AUTHORIZED_DOMAIN_STATUS_OK = 'ok';
+  const AUTHORIZED_DOMAIN_STATUS_ERROR = 'authorized_domain_error';
   const SENDING_STATUS_OK = 'ok';
   const SENDING_STATUS_CONNECTION_ERROR = 'connection_error';
   const SENDING_STATUS_SEND_ERROR = 'send_error';
```

Comment:
Just an idea. Do we need too many special `ok` statuses? I mean the caller knows which method was called so maybe we could live with a common `ok` status shared by all endpoints. We could also have a common error status shared by all endpoints + a couple of special error statuses if needed.

I think we could change:
```php
  const AUTHORIZED_EMAIL_STATUS_OK = 'ok';
  const AUTHORIZED_EMAIL_STATUS_ERROR = 'authorized_email_error';
  const AUTHORIZED_DOMAIN_STATUS_OK = 'ok';
  const AUTHORIZED_DOMAIN_STATUS_ERROR = 'authorized_domain_error';
  const SENDING_STATUS_OK = 'ok';
  const SENDING_STATUS_CONNECTION_ERROR = 'connection_error';
  const SENDING_STATUS_SEND_ERROR = 'send_error';
```

to:
```php
  const RESPONSE_STATUS_OK = 'ok';
  const RESPONSE_STATUS_ERROR = 'error';
  const SENDING_STATUS_CONNECTION_ERROR = 'connection_error';
  const SENDING_STATUS_SEND_ERROR = 'send_error';
```

Reply:
I will change it. Good idea. I was considering something similar but didn't have the final idea.